garglk/cheapglk/cgdate.cpp

```diff
@@ -176,32 +179,47 @@ static glsi32 gli_simplify_time(time_t timestamp, glui32 factor)
 
 void glk_current_time(glktimeval_t *time)
 {
+#ifndef WIN32
```

Comment:
My understanding is that `_WIN32` is defined by the compiler, and `WIN32` by headers, so that `_WIN32` is a bit less brittle; unless there's a strong reason to use `WIN32`, I'd prefer a blanket usage of `_WIN32` for consistency and robustness.